data/json/items/book/maps.json

```diff
@@ -77,6 +77,20 @@
       "message": "You add roads and points of interest to your map."
     }
   },
+  {
+    "id": "subwaymap",
+    "copy-from": "abstractmap",
+    "type": "GENERIC",
+    "name": "subway map",
+    "description": "This is a map of subway routes and stations in the area. Using it will add subway lines and underground stations to your map.",
```

Comment:
```suggestion
    "description": "This is a map of subway routes and stations in the area.  Using it will add subway lines and underground stations to your map.",
```